_projects/sssp_project.md

```diff
@@ -41,14 +41,18 @@ We consider a machine learning methodology to give appropriate weighting factors
 
 We have proposed the idea of Simpelified SSP and evaluated its consistency with the original SSP over several systems. 
 
+## Results for 2017/2018
+
+We have performed 6 tnraditional benchmarks and 7 mini applications on 6 systems to calculate the SSP and SSSP metrics.
+The SSSP metric has given better performance projection to the SSP than HPL. However, there have been still some differences between the SSP and SSSP scores. Therefore, we have introduced weighting factors for benchmarks in the SSSP to approximate the SSP metrics by the SSSP matrics more acurately. The weighting factors have been calculated based on a simple learning algorithm, and the SSSP metric using the resulted weighting factors has succcessfully approximated the SSP metric. 
+
 ## Visits and meetings
 
 Miwako Tsuji visited NCSA in January 2017.
 
 ## Impact and publications
 
-None yet.
-
+Miwako Tsuji, William T. C. Kramer and Mitsuhisa Sato,"A Performance Projection of Mini-Applications onto Benchmarks Toward the Performance Projection of Real-Applications", 2017 IEEE International Conference on Cluster Computing (CLUSTER) Workshop on Representative Applications (WRAp)
 
```

Comment:
Please use the bib files for citing/referencing, in particular for JLESC papers. See [jlesc.bib](https://github.com/JLESC/jlesc.github.io/blob/source/_bibliography/jlesc.bib) for the bib file and e.g. [this project](https://jlesc.github.io/projects/ichpc_project/) for an example.